README.md

```diff
@@ -41,6 +41,14 @@ You can find out more existing tools & documentation for Gno on our
 [awesome-gno](https://github.com/gnolang/awesome-gno) repository.
 We look forward to seeing your first PR!
 
+## Gno Playground
+<a href="https://play.gno.land/p/VxDC6AmKmK6?run.expr=println(Render(%22%22))">
```

Comment:
I suggest we switch to the markdown syntax.

Reply:
@salmad3